TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -461,8 +463,7 @@ public long SlackBytes
                         //Get the original redefined node
                         while (redefines != null)
                         {
-                            SymbolReference redefined = redefines.CodeElement.RedefinesDataName;
-                            redefinedDataDefinition = redefines.SymbolTable.GetRedefinedVariable(redefines, redefined);
+                            redefinedDataDefinition = redefines.RedefinedVariable;
 
                             redefines = redefinedDataDefinition as DataRedefines;
                         }
```

Comment:
This is not needed anymore as `RedefinedVariable` is now transitive

Reply:
Loop removed